tests/unit/dataactvalidator/test_scripts_loadTas.py

```diff
@@ -45,3 +51,38 @@ def test_loadTas_space_nulls(database, tmpdir):
     assert results['beginning_period_of_availability'][0] is None
     assert results['ending_period_of_availability'][0] is None
     assert results['availability_type_code'][0] is None
+
+
+def test_updateTASLookups(database, monkeypatch):
+    sess = database.session
+    monkeypatch.setattr(
+        loadTas, 'cleanTas', Mock(return_value=pd.DataFrame({
+            'availability_type_code': ['0', '1', '2'],
+            'tas_id': [111, 222, 333],
+            'agency_identifier': ['0', '1', '2'],
+            'allocation_transfer_agency': ['0', '1', '2'],
+            'beginning_period_of_availability': ['0', '1', '2'],
+            'ending_period_of_availability': ['0', '1', '2'],
+            'main_account_code': ['0', '1', '2'],
+            'sub_account_code': ['0', '1', '2'],
+        }))
+    )
+
+    sess.add(TASFactory(tas_id=222))
+    sess.add(TASFactory(tas_id=444, agency_identifier='other'))
+    sess.commit()
+
+    results = sess.query(TASLookup).order_by(TASLookup.tas_id).all()
+    assert len(results) == 2
+    sess.invalidate()
+
+    loadTas.updateTASLookups('ignored')
```

Comment:
Thank you for entertaining my various unit testing questions. What does the `'ignored'` do here? Is that a signal to code readers that `updateTASLookups` has been monkey patched?